feature-utils/poly-look/src/css/layout.css

```diff
@@ -10,3 +10,8 @@
 .poly-standard-layout > :first-child {
   margin-top: calc(var(--gdu) * 6);
 }
+
+.poly-standard-layout h2 {
+  font-family: var(--jost-medium);
+  font-size: var(--font-size-xl);
```

Comment:
We sure about this? This would be the biggest size, right?

Reply:
If this substitutes this other:

https://github.com/polypoly-eu/polyPod/pull/988/files#diff-e84144b2d0596c36f283b5461d52fd1147f9b890694a6c12f686a8503f3fc338L16-L21

Are you changing h1 → h2?